src/themes/shared/zIndex.ts

```diff
@@ -0,0 +1,8 @@
+export const zIndices = {
```

Comment:
are those coming from our tokens? I find the keys confusing because I might want to use 1500 for something else than a menu. Also, is there a reason the menu should be in front of the modal?

I would suggest to use the actual numbers as the keys if that is in-line with our tokens. Also, another reason to not use named keys would be the context in which they are used. If you are in a fixed or sticky container for example, `overlay` might require another value

Reply:
yea designer will add them to the tokens.
I agree with names, better to use numbers as names, will align it with designer as well.

Reply:
So as discussed with designer in slack for now we will use chakra indicies https://chakra-ui.com/docs/styled-system/theme#z-index-values and we can adopt them with semantic tokens if needed.